float32 baro_millibars
float32 temp_C
float32 wind_mph
float32 wind_direction
